generator/metaline_builder.cpp

```diff
@@ -0,0 +1,217 @@
+#include "generator/metaline_builder.hpp"
+#include "generator/routing_helpers.hpp"
+
+#include "indexer/classificator.hpp"
+
+#include "coding/file_container.hpp"
+#include "coding/varint.hpp"
+#include "coding/write_to_sink.hpp"
+
+#include "base/checked_cast.hpp"
+#include "base/logging.hpp"
+#include "base/string_utils.hpp"
+
+#include "defines.hpp"
+
+#include <cstdint>
+
+namespace
+{
+using Ways = std::vector<std::int32_t>;
```

Comment:
So, if you afraid that at some point in the future the standard header cstdint will stop to expose all these types to the global namespace, write ```using namespace std;``` just below the block of includes. In our project it's a common practice to include cstdint and not writing "std::uint64_t" - please, take a look at the code around. I'm suspecting we spend a lot of time on the useless discussion.